app/models/user.rb

```diff
@@ -6,7 +6,11 @@ class User < ActiveRecord::Base
   has_many :neighborhoods, through: :user_neighborhoodss
 
   has_many :user_groups
-  has_many :groups, through: :user_groups
+  has_many :groups, through: :user_groups do
```

Comment:
Using this scope, we're able to do `current_user.groups.groups_led` and it gives us the list of groups that current user is leading @andylouisqin @HacksForHugs @nikitr